packages/@react-aria/tooltip/test/useTooltip.test.js

```diff
@@ -10,18 +10,211 @@
  * governing permissions and limitations under the License.
  */
 
+import {act, fireEvent, render} from '@testing-library/react';
 import React from 'react';
-import {renderHook} from '@testing-library/react-hooks';
-import {useTooltip} from '../';
-
-describe.skip('useTooltip', function () {
-  let renderTooltipHook = (props) => {
-    let {result} = renderHook(() => useTooltip(props));
-    return result.current;
-  };
-
-  it('handles defaults', function () {
-    let {tooltipProps} = renderTooltipHook({children: 'Test Tooltip'});
-    expect(tooltipProps.role).toBe('tooltip');
+import {useInteractionModality} from '@react-aria/interactions';
+import {useTooltip, useTooltipTrigger} from '../';
+import {useTooltipTriggerState} from '@react-stately/tooltip';
+
+describe('useTooltip', function () {
+  afterEach(() => {
+    jest.useRealTimers();
+  });
+  it('opens tooltip on hover after delay', function () {
+    jest.useFakeTimers('modern');
+
+    function Test() {
+      useInteractionModality();
+
+      const state = useTooltipTriggerState();
+      const {triggerProps, tooltipProps} = useTooltipTrigger({}, state);
+      const {tooltipProps: finalTooltipProps} = useTooltip(tooltipProps);
+
+      return (
+        <div>
+          <span {...triggerProps}>{'Trigger'}</span>
+          {state.isOpen && <span {...finalTooltipProps}>{'Tooltip'}</span>}
+        </div>
+      );
+    }
+
+    const {container, queryByRole} = render(<Test />);
+    const triggerElement = container.firstChild.firstChild;
+
+    // trigger pointer modality
+    fireEvent.mouseMove(container);
```

Comment:
OTOH having to do this sounds OK - I've scratched my head a little bit before I've realized that I have to do that (because `mouseenter` is not handled within `setupGlobalFocusEvents`) but in the real apps `mousemove` will always occur before `mouseenter` on the trigger element so I don't think that changing the implementation to cover testing quirks is required.

Reply:
yeah, might be good to have a util for this eventually, but we don't need to do it here